pkg/trimaran/targetloadpacking/targetloadpacking.go

```diff
@@ -79,15 +75,21 @@ func New(obj runtime.Object, handle framework.FrameworkHandle) (framework.Plugin
 	}
 	hostTargetUtilizationPercent = args.TargetUtilization
 	requestsMilliCores = args.DefaultRequests.Cpu().MilliValue()
-	watcherAddress = args.WatcherAddress
 	requestsMultiplier, _ = strconv.ParseFloat(args.DefaultRequestsMultiplier, 64)
 
 	podAssignEventHandler := trimaran.New()
+
+	var metricclient loadwatcherapi.Client
+	if args.MetricProviderType == LoadWatcherServiceClientName {
```

Comment:
This is not needed -> one wouldn't set `args.MetricProviderType` if using watcher as a service, only `watcherAddress` would be configured.

Reply:
Replaced as `if args.watcherAddress != ""`